src/AdminConsole/Components/Pages/App/Settings/SettingsComponents/AuthenticationConfiguration.razor

```diff
@@ -0,0 +1,117 @@
+@using System.Collections.Immutable
+@using Humanizer
+@using Passwordless.AdminConsole.Helpers
+@using Passwordless.Common.Models.Apps
+@inject ICurrentContext CurrentContext;
+@inject NavigationManager NavigationManager;
+@inject IHttpContextAccessor HttpContextAccessor;
+@inject IScopedPasswordlessClient Passwordless;
+
+@if (!CurrentContext.IsPendingDelete)
+{
+    <Panel Header="Authentication Configuration">
+        <p>List of configurations for your authentication tokens. Read more in the <a class="link-blue underline" href="https://docs.passwordless.dev/guide/admin-console/applications.html#authentiation-configurations">docs</a></p>
+
+        <div>
+            <a class="btn-primary" href="/app/@(AppId)/settings/authentication-configuration/new">Add New Authentication Configuration</a>
+        </div>
+
+        <Table ColumnHeaders="@(new[] { "Purpose", "User Verification", "Time To Live", "Last Used", "Actions" })">
+            @if (Configurations.Any())
+            {
+                @foreach (var config in Configurations)
+                {
+                    <tr>
+                        <td>@config.Purpose</td>
+                        <td>@config.UserVerification</td>
+                        <td>@config.TimeToLive</td>
+                        <td>@config.LastUsedOn?.ToString("g")</td>
+                        <td class="flex items-center space-x-2">
+                            <a href="/app/@(AppId)/settings/authentication-configuration/@(config.Purpose)/edit">
+                                <PencilSquareIcon Class="h-6 w-6"></PencilSquareIcon>
+                            </a>
+
+                            @if (config.CanDelete)
+                            {
+                                <ConfirmEditForm Model="ConfirmDeleteAuthenticationConfigurationModel" FormName="@nameof(ConfirmDeleteAuthenticationConfigurationModel)" name="@nameof(ConfirmDeleteAuthenticationConfigurationModel)" OnSubmit="OnSelectedFormConfirmed">

```

Comment:
I don't think this was working for regular forms to put a form with the same name in every cell. This can have unintended consequences later.

You can wrap the entire table in a `ConfirmForm`. And then for `ConfirmButton`, set the purpose as an attribute of the HTML element. Then add ConfirmButton name attribute and set that to action for example with value attribute set to `delete` if you plan on supporting multiple actions later.

Reply:
I did something a little different for this fix. Let me know what you think.

Reply:
Transitioned to 1 form for the whole list to get rid of the need for a hidden field. However, still couldn't figure out the model binding for post back.  I don't know if its because its a blazor component embedded on a razor page, but just manually passing the value now.

Reply:
In #412, I have described that model binding is broken, including the handling of the form submission events. The `ApiKeysSection` component contains an example how to check when your component runs on a Razor Page and when you have a form submission at the same time.